app/src/androidTest/java/com/github/bgabriel998/softwaredevproject/points/ComputePOIPointsTest.java

```diff
@@ -24,11 +24,14 @@
     @Rule
     public GrantPermissionRule grantCameraPermissionRule = GrantPermissionRule.grant(Manifest.permission.CAMERA);
     private static UserPoint userPoint;
+    private static Context mContext;
 
     @BeforeClass
     public static void setUserPoint(){
         //Set userPoint to the mont blanc
-        userPoint = new UserPoint(45.802537, 6.850328, 4809);
+        mContext = ApplicationProvider.getApplicationContext();
+        userPoint = UserPoint.getInstance(mContext); // TODO check if working properly
+        userPoint.setLocation(46.519251915333676, 6.558563221333525, 220, 0);
```

Comment:
You could rechange location to the mont-blanc and change the altitude or maybe change the comment above